seqr/views/apis/report_api_tests.py

```diff
@@ -1293,7 +1296,9 @@ def test_variant_metadata(self):
             **BASE_VARIANT_METADATA_ROW,
             'alt': None,
             'chrom': '1',
-            'condition_id': 'MONDO:0008788',
+            'condition_id': 'OMIM:616126',
```

Comment:
test the ability to report OMIM for one variant in a family and MONDO for another - the above test for `['rows'][5]` still reports the Modno condition but this row now reports in OMIM